src/mixins/render-vega-lite/Definitions/Config/ConfigDefinitionInterface.js

```diff
@@ -0,0 +1,32 @@
+import PropertiesDefinitionInterface from "../PropertiesDefinitionInterface"
+
+export default class ConfigDefinitionInterface extends PropertiesDefinitionInterface {
+  /**
+   * @param {PropDescriptor} prop_descriptor
+   * @param {VegaPropertyOutputState} vega_property_output_state
+   */
+  materializeProperty(prop_descriptor, vega_property_output_state) {
```

Comment:
As a general thing, javascript convention is to use camel case. Probably too much pain to convert everything you've done so far but for future things

Reply:
sorry, to use camel case for what, the argument names, like `prop_descriptor`? 

Ok, does the FE have any formal coding style guides? I don't think I've ever seen any. This style is coming out of the BE side of things where class/struct/API methods are camel case, arguments and free functions are snake case. It's a form I'm used to and therefore prefer (with bias).

Reply:
To your question, yes, all variable and function names are generally camel case.

Makes sense with your background, just pointing out that it is not the norm in javascript development. Variable and function names are camel case, class names upper camel case is the norm. Javascript itself uses this convention:
<img width="437" alt="Screen Shot 2022-11-21 at 11 58 17 AM" src="https://user-images.githubusercontent.com/1031659/203137331-7a437c72-cbb7-4b28-a1db-04cfa1a74663.png">

There's no heavy style guide as far as I know, but definitely not a bad idea.

Reply:
I guess that only shows function names, but take my word for it 😉

Reply:
Also just to be clear, not asking for change, snake case def works and I've seen it before but I think we should use camel case going forward since I think this is more the norm + what we do in other heavy repos (mainly immerse, connector too)

@cmatzenbach any thoughts?